darwin/Platform.c

```diff
@@ -139,6 +139,8 @@ void Platform_init(void) {
 
    const double nanos_per_sec = 1e9;
    Platform_nanosecondsPerSchedulerTick = nanos_per_sec / scheduler_ticks_per_sec;
+
+   return 0;
```

Comment:
This return value doesn't make much sense, in particular as an int. IMHO `bool` (`stdbool.h`) would be better suited.